### ISSUE TITLE
To-Do List and Updates

### ISSUE THREAD

**Issue by supercoolsonic** [supercoolsonic/SADXDC-SET-Layout-Fixes#6]
This is where I'll post things that I want to get done for the SET Layout Fixes mod, as well as anything that happens irl to me that will impact the progress of work on the mod.

I won't be releasing any more fixes until I finish at least most of this list.

TODO:
-Dumb spring in Big's Hot Shelter Act 1.
-Sky Deck Act 1 automated section.
-Rare problem in Sky Deck Act 3 where a few dash panels will miss.
-Collision problem with a spring in Windy Valley Act 2, the same that SF94 mentioned in the original SADXFE project.
-Include any fixes made to Sonic's Emerald Coast Act 2 in Tails' file (Maybe?).
-Implement a Super Sonic friendly fix for the Speed Highway Act 1 corkscew (Maybe? He misses the last ring in the line usually with the current version.).
-Implement some way to prevent Super Sonic from going flying during that long spring path in Windy Valley Act 3 (Maybe? Might take more work than simple rotation.).
-Lots more things to do in Gamma's Hot Shelter.
-Poke around Sandboard minigame to see if anything is awry.
-Perhaps optimize Boss fights? Many of them have rings out of bounds.
-Examine levels and possibly remove erroneous objects at the origin (0,0,0).
-Take a closer look at that one Buyon in Twinkle Park Act 3.
-First Ghola in Sonic's Lost World Act 2 sometimes fails to spawn?
-Look at the positioning for the door that separates the train room and the casino area in Station Square.
-Collision problem on lower part of transformed Egg Carrier.

(On standby to be added to SET Fixes)
-Collision problem around wind gust in Mystic Ruins that takes you to Angel Island. 
-OTTOTTO objects around the top of Twinkle Castle should not slow you down when you get past the roof segment.
-Collision problems with certain sections of the wall at the top of Twinkle Castle.

Finished, but not yet added to official layouts (released):
Collision fix for Sonic's Twinkle Park Act 2 at the top right of the stairs.

May 25th - I'm going to be gone for a little over a week starting tomorrow, and will be without access to my computer. I will resume work on the mod when I return.
In terms of working with other things, I've been going at the Sky Deck automated section problem and dash panel problem for a while now, and dear god, these are a real pain in the ass. The automated section just does _not_ want to cooperate, and it's annoying to test the dash panels because they require so much testing. The issue is rare, so each change I make I have to test it, like, 50 times. But I'm making progress. I _will_ get these to work properly.

**Comment by supercoolsonic** [comment]
June 3rd - I have returned to my living quarters and now have access to my computer again. I will state the exact progress on the Sky Deck automated section. I have it working fairly well, but there's this stupid problem where Sonic and Tails like to abruptly stop for no apparent reason at the start of the PATH...path. Then there's also the problem where the scripted sequence goes on further than it should. When you hit that spring at the end of the segment, the platform it takes you to is still under the influence of the PATH data, which means moving too much will lead you suddenly blasting away off the platform. I've placed OTTOTTO objects around the edges of the platform to try to remedy it, but that's a blatant coding issue. That needs to be fixed with actual coding. I will resume work on this tomorrow.
